src/backup.sh

```diff
@@ -83,3 +83,13 @@ aws $aws_args s3 cp "$local_file" "$s3_uri"
 rm "$local_file"
 
 echo "Backup complete."
+
+if [ "$BACKUP_KEEP_DAYS" -ne 0 ]; then
+  ms=$((86400*BACKUP_KEEP_DAYS))
+  date_from_remove=$(date -d "@$(($(date +%s) - ms))" +%Y-%m-%d)
+  backups_query="Contents[?LastModified<='${date_from_remove} 00:00:00'].{Key: Key}"
+
+  echo "Removing old backup from $S3_BUCKET..."
+  aws s3api list-objects --bucket "${S3_BUCKET}" --prefix "${S3_PREFIX}" --query "${backups_query}" --output text | xargs -n1 -t -I 'KEY' aws s3 rm s3://"${S3_BUCKET}"/'KEY'
```

Comment:
Please use the --long-form xargs flags for readability (I'll admit, I had to look all these up :laughing:). Also a line break or two if you don't mind.

Reply:
For some reason long-form `xargs` doesn't work with the Alpine container(